pyfar/signals/signals.py

```diff
@@ -0,0 +1,568 @@
+    nl = "\n"  # required as variable because f-strings cannot contain "\"
+    comment = f"White noise signal (rms = {str(rms).replace(nl, ',')})"
+
+    signal = Signal(noise, sampling_rate, fft_norm="rms", comment=comment)
+
+    return signal
+
+
+def pink_noise(n_samples, rms=1, sampling_rate=44100, seed=None):
+    """Generate single or multi channel normally distributed pink noise.
+
+    The pink noise is generated by applying a sqrt(1/f) filter to the spectrum.
+
+    Parameters
+    ----------
+    n_samples : int
+        The length of the signal in samples
+    rms : double, array like, optional
+        The route mean square (RMS) value of the noise signal. A multi channel
+        noise signal is generated if an array of RMS values is passed.
+        The default is 1.
+    sampling_rate : int, optional
+        The sampling rate in Hz. The default is 44100.
+    seed : int, None
+        The seed for the random generator. Pass a seed to obtain identical
+        results for multiple calls of white noise. The default is None, which
+        will yield different results with every call.
+
+    Returns
+    -------
+    signal : Signal
+        The noise as a Signal object. The Signal is in the time domain and
+        has the 'rms' FFT normalization (see pyfar.fft.normalization).
+    """
+
+    # generate the noise
+    rms = np.atleast_1d(rms)
+    noise = _generate_white_noise(n_samples, rms, seed)
+
+    # apply 1/f filter in the frequency domain
+    noise = fft.rfft(noise, n_samples, sampling_rate, 'none')
+    noise /= np.sqrt(np.arange(1, noise.shape[-1]+1))
+    noise = fft.irfft(noise, n_samples, sampling_rate, 'none')
+
+    # level the noise
+    noise = _normalize_rms(noise, rms)
+
+    # save to Signal
+    nl = "\n"  # required as variable because f-strings cannot contain "\"
+    comment = f"Pink noise signal (rms = {str(rms).replace(nl, ',')})"
+
+    signal = Signal(noise, sampling_rate, fft_norm="rms", comment=comment)
+
+    return signal
+
+
+def pulsed_noise(n_pulse, n_pause, n_fade=90, repetitions=5, rms=1,
+                 color="pink", frozen=True, sampling_rate=44100, seed=None):
+    """
+    Generate single channel normally distributed pulsed white and pink noise.
+
+    The pink noise is generated by applying a sqrt(1/f) filter to the spectrum.
+
+    Parameters
+    ----------
+    n_pulse : int
+        The length of the pulses in samples
+    n_pause : int
+        The length of the pauses between the pulses in samples.
+    n_fade : int, optional
+        The length of the squared sine/cosine fade-in and fade outs in samples.
+        The default is 90, which equals approximately 2 ms at sampling rates of
+        44.1 and 48 kHz.
+    repetitions : int, optional
+    rms : double, array like, optional
+        The RMS amplitude of the white noise signal. The default is 1.
+    color: string, optional
+        The noise color, which can be 'pink' or 'white'. The default is 'pink'.
```

Comment:
I think color is misleading here, since we're talking about audio signals and not light. I would rename this to for example `spectrum`, since it's the spectral coloration of our noise signal we're defining here. Makes it a tad more explicit.